legend-engine-executionPlan-execution-store-relational/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/manager/strategic/DataSourceIdentifiersCaseSensitiveVisitor.java

```diff
@@ -13,6 +14,11 @@ public Boolean visit(DatasourceSpecification datasourceSpecification)
             SnowflakeDatasourceSpecification snowflakeDatasourceSpecification = (SnowflakeDatasourceSpecification) datasourceSpecification;
             return snowflakeDatasourceSpecification.quotedIdentifiersIgnoreCase == null || !snowflakeDatasourceSpecification.quotedIdentifiersIgnoreCase;
         }
+        else if (datasourceSpecification instanceof RedshiftDatasourceSpecification)
+        {
+//            documentaion link : https://docs.aws.amazon.com/redshift/latest/dg/r_names.html
```

Comment:
Nitpick. Typo in "documentaion"

Reply:
@davidharte-gs redshift has a flag `enable_case_sensitive` which is by default false, should we specify this explicitly so that users can set this true in rare case.
Since its default as false and if we use this logic it would work in both scenario's